### PR TITLE
Added JT4 models

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/xxx

## Summary

This PR adds the new JT4 models for Biometric Enrollment

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: Sources/SmileID/Classes/Networking/Models/v2/JobType.swift .swift
Comment (vanshg) [inline Sources/SmileID/Classes/Networking/Models/v2/JobType.swift .swift:1]: Check the filename here. You have `.swift` twice

File: Sources/SmileID/Classes/Networking/Models/v2/JobType.swift .swift [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+public enum JobTypeV2: String, Codable {
```
Comment (vanshg) [inline Sources/SmileID/Classes/Networking/Models/v2/JobType.swift .swift:3]: Is it possible to name this just JobType? Or would this cause conflicts

Comment (jumaallan) [reply Sources/SmileID/Classes/Networking/Models/v2/JobType.swift .swift:3]: Causes conflicts

**Review by vanshg — APPROVED** [review]
